docs/content/concepts/resources.mdx

```diff
@@ -260,3 +260,10 @@ When constructing a job that includes that op, we provide the resource `client`,
 def connect():
     get_client()
 ```
+
+## See it in action
+For more examples of resources, check out the following in our [Hacker News example](https://github.com/dagster-io/dagster/tree/master/examples/hacker_news):
+* [Hacker News resource](https://github.com/dagster-io/dagster/blob/master/examples/hacker_news/hacker_news/resources/hn_resource.py)
+* [dbt asset resource](https://github.com/dagster-io/dagster/blob/master/examples/hacker_news/hacker_news/resources/dbt_asset_resource.py)
```

Comment:
I think this one is a bit confusing and might be better left out.

Reply:
just confirming, you think the dbt asset resource is confusing?

Reply:
exactly